src/components/application/index.js

```diff
@@ -1,23 +1,365 @@
+import _ from 'lodash';
```

Comment:
we should look into using lodash modules instead of the full package, especially since this is client side code. https://www.npmjs.com/browse/keyword/lodash-modularized

Reply:
or just do `import isEmpty from 'lodash/isEmpty`, etc